Rick&Morty/RickAndMorty/Application/Resources/PreviewAssets/CharactersMock.swift

```diff
@@ -0,0 +1,704 @@
+//
+//  CharactersMock.swift
+//  RickAndMorty
+//
+//  Created by Róbert Oravec on 05.08.2022.
+//  Copyright © 2022 STRV. All rights reserved.
+//
+
+import Foundation
+
+enum CharactersMock {
```

Comment:
Is this really necessary? I don't like the fact that because of this simple mock you brought also Decodable to the codebase even though you can simply use just `[Character.mock, Character.mock, ...]` or if you really want to have more characters then you can create a few more manually 🙏